extras/docker/base/Dockerfile

```diff
@@ -6,7 +6,11 @@ MAINTAINER Counterparty Developers <dev@counterparty.io>
 RUN apt-get update && apt-get install -y apt-utils ca-certificates wget curl git mercurial \
     python3 python3-dev python3-pip python3-setuptools python3-appdirs \
     build-essential vim unzip software-properties-common sudo gettext-base \
-    net-tools iputils-ping telnet lynx
+    net-tools iputils-ping telnet lynx nano
+
+# @TODO: commut jq-linux64 to this repo?
+RUN wget https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
+RUN sudo mov jq-linux64 /usr/bin/jq; sudo chmod +x /usr/bin/jq
```

Comment:
easy debugging tool for JSON, I think we should commit the binary into the repo instead of downloading from untrusted source